components/org.wso2.carbon.identity.media.endpoint/src/main/java/org/wso2/carbon/identity/media/endpoint/service/MediaService.java

```diff
@@ -314,4 +323,134 @@ private void validateAllowedMediaTypes(String mediaTypePathParam) {
         }
     }
 
+    /**
+     * Delete a media.
+     *
+     * @param type The high level content-type of the resource (if media content-type is image/png then
+     *             type would be image.
+     * @param id   Unique identifier for the requested media file to be deleted.
+     */
+    public void deleteMedia(String type, String id) {
+
+        validateAllowedMediaTypes(type);
+        StorageSystemManager storageSystemManager = getStorageSystemManager();
+        String tenantDomain = getTenantDomainFromContext();
+        String mediaStoreType = getMediaStoreType();
+        try {
+            boolean mediaDeleted = storageSystemManager.isMediaDeleted(id, type, tenantDomain, mediaStoreType);
```

Comment:
does isMediaDeleted responsible for deleting the media as well? if so, let's change the method name to be more meaningful

Reply:
yes it is responsible for deleting media. Is it ok to change the name to `deleteMedia` ?